### PR TITLE
Created web content filtering events query

### PR DESCRIPTION
**Pull request by mjmelone**
Required items, please complete
   
   Change(s):
   - New query created

   Reason for Change(s):
   - No preexisting query for WCF events

   Version Updated:
   - Set version to 1

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Validations have passed.

### CODE REVIEW

File: Hunting Queries/Microsoft 365 Defender/General queries/Web Content Filtering Events.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+id: 7f7a796d-1511-4930-b9da-5971db4352ec
+name: Web Content Filtering Events
+version: 1
```
Comment (aprakash13) [inline Hunting Queries/Microsoft 365 Defender/General queries/Web Content Filtering Events.yml:3]: I don't think we need the version field here.

File: Hunting Queries/Microsoft 365 Defender/General queries/Web Content Filtering Events.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+description: |
+  This query identifies web content filtering events in Advanced Hunting.
+requiredDataConnectors:
+- connectorId: Microsoft365Defender
```
Comment (aprakash13) [inline Hunting Queries/Microsoft 365 Defender/General queries/Web Content Filtering Events.yml:7]: Please change the connector to MicrosoftThreatProtection

File: Hunting Queries/Microsoft 365 Defender/General queries/Web Content Filtering Events.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  dataTypes:
+  - DeviceEvents
```
Comment (aprakash13) [inline Hunting Queries/Microsoft 365 Defender/General queries/Web Content Filtering Events.yml:9]: Please add Tactics field as well.

**Review by aprakash13 — CHANGES_REQUESTED** [review]

**Comment by shainw** [comment]
@mjmelone - Thanks for the contribution, please review requested changes.

**Comment by vmanojreddy** [comment]
@mjmelone please update the requested changes ,thanks!!!

**Comment by vmanojreddy** [comment]
@mjmelone please update the requested changes ,thanks!!!

**Comment by mjmelone** [comment]
@vmanojreddy \ @shainw updated.

**Comment by vmanojreddy** [comment]
@aprakash13 Please check and approve the request changes. thanks!!!

**Comment by vmanojreddy** [comment]
@aprakash13 Please check and approve the request changes. thanks!!!

**Pull request merged**